frontend/src/screens/settings/Backup.tsx

```diff
@@ -0,0 +1,460 @@
+    e.preventDefault();
+
+    const sixMonthsLater = new Date();
+    sixMonthsLater.setMonth(sixMonthsLater.getMonth() + 6);
+
+    try {
+      await request("/api/backup-reminder", {
+        method: "PATCH",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          nextBackupReminder: sixMonthsLater.toISOString(),
+        }),
+      });
+      await refetchInfo();
+      navigate("/");
+      toast({ title: "Recovery phrase backed up!" });
+    } catch (error) {
+      handleRequestError(toast, "Failed to store back up info", error);
+    }
+  }
+
+  return (
+    <>
+      <SettingsHeader
+        title="Backup"
+        description={
+          <>
+            <span className="text-muted-foreground">
+              Backup your wallet recovery phrase and or your channel states in
+              order to migrate your node.{" "}
+            </span>
+            <a
+              href="https://guides.getalby.com/user-guide/alby-account-and-browser-extension/alby-hub/backups-and-recover"
+              target="_blank"
+              rel="noreferrer noopener"
+              className="text-foreground underline"
+            >
+              {" "}
+              Learn more about backups
+            </a>
+          </>
+        }
+      />
+
+      {hasMnemonic && (
+        <div className="flex flex-col gap-6">
+          <div>
+            <h3 className="text-lg font-medium">Wallet Keys Backup</h3>
+            <p className="text-sm text-muted-foreground">
+              Key recovery phrase is a group of 12 random words that back up
+              your wallet on-chain balance. Using them is the only way to
+              recover access to your wallet on another machine or when you loose
+              your unlock password.
+            </p>
+          </div>
+          <p className="text-destructive">
+            If you loose access to your Hub and do not have your recovery
+            phrase, you will loose access to your funds.
+          </p>
+          {info?.backendType === "CASHU" && <CashuMnemonicWarning />}
+
+          <div>
+            <form
+              onSubmit={onSubmitPassword}
+              className="max-w-md flex flex-col gap-3"
+            >
+              <div className="grid gap-2 mb-6">
+                <Label htmlFor="password">Password</Label>
+                <PasswordInput id="password" onChange={setUnlockPassword} />
+                <p className="text-sm text-muted-foreground">
+                  Enter your unlock password to view your recovery phrase.
+                </p>
+              </div>
+              <div className="flex justify-start">
+                <LoadingButton loading={loading} variant="secondary" size="lg">
+                  View Recovery Phrase
+                </LoadingButton>
+              </div>
+            </form>
+          </div>
+
+          <Dialog open={isDialogOpen} onOpenChange={setIsDialogOpen}>
```

Comment:
Could we move this component to a separate file?

Also it doesn't seem to fit so well on a standard screen. Also on mobile I don't think it's nice to have a scrollbar inside a modal. @stackingsaunter do you have any thoughts?

![image](https://github.com/user-attachments/assets/8ae928b8-640f-44ba-80c4-70d1d9a93434)

![image](https://github.com/user-attachments/assets/a8f61455-c89c-44d2-836a-3af2771603c7)

Reply:
already moved it to separate file. let me know if anything additional is to be done. resolving for now

Reply:
@pavanjoshi914 I clearly mentioned about the scrollbar and there was no action taken. Please do not resolve issues that are not resolved. We will miss things that were already reviewed.

Reply:
sry! i guess i moved it to separate file and forgot about scrolll issue. added new no-scrollbar classes